### PR TITLE
Completed issue#1

### PR DESCRIPTION
**Pull request by JimmyTran1206**
https://github.com/JimmyTran1206/ajax-project/assets/67088234/23ff00c6-5101-46c3-8492-6c3e64552eb1

### CODE REVIEW

**Review by shawnkost — CHANGES_REQUESTED** [review]
Few changes requested here Jimmy!

File: index.html [hunk 1]
```diff
@@ -8,12 +8,110 @@
...
+            <div class="content-title">
+              <h3 class="anime-title"></h3>
+            </div>
+            <div class="content-image">
+              <img class="anime-image" src="" alt="">
+            </div>
+
+          </div>
+        </div> -->
+        <!-- ......................... -->
```
[the remark this answers is not in the document]
Comment (JimmyTran1206) [reply index.html:78]: corrected

File: index.html [hunk 2]
```diff
@@ -8,12 +8,110 @@
...
+                    <div class="content-title">
+                      <h3 class="anime-title"></h3>
+                    </div>
+                    <div class="content-image">
+                      <img class="anime-image" src="" alt="">
+                    </div>
+
+                  </div>
+                </div> -->
+        <!-- ......................... -->
```
[the remark this answers is not in the document]
Comment (JimmyTran1206) [reply index.html:95]: corrected

File: index.html [hunk 3]
```diff
@@ -8,12 +8,110 @@
...
+      </div>
+
+      <div class="container-error hidden">
+        <div style="width:85%" class="content-box">
```
[the remark this answers is not in the document]
Comment (JimmyTran1206) [reply index.html:99]: corrected

File: index.html [hunk 4]
```diff
@@ -8,12 +8,110 @@
...
+          <div class="content-item" id="Error">
+            <div class="content-title">
+              <h3 style="font-size: 48px;" class="anime-title">おっと Oops, some errors has occurred. Please try again later. Thank you. ありがとう</h3>
```
[the remark this answers is not in the document]
Comment (JimmyTran1206) [reply index.html:102]: corrected

File: index.html [hunk 5]
```diff
@@ -8,12 +8,110 @@
...
+            </div>
+            <div class="content-image">
+              <img class="anime-image" src="images/anime-girls-404-not-found.jpg" alt="">
```
[the remark this answers is not in the document]
Comment (JimmyTran1206) [reply index.html:105]: corrected

File: js/main.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+// show-hide container function
+function showContainer($domContainer) {
+  $domContainer.classList.remove('hidden');
+}
+function hideContainer($domeContainer) {
+  $domeContainer.classList.add('hidden');
+}
+// fetch-display function for seasonal-popular animes
+function fetchDisplay(url, $domElem) {
+  $domElem.innerHTML = '';
```
[the remark this answers is not in the document]
Comment (JimmyTran1206) [reply js/main.js:46]: corrected

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**